packages/wonder-blocks-form/src/components/labeled-text-field.js

```diff
@@ -83,7 +83,7 @@ type Props = {|
     light: boolean,
 
     /**
-     * Custom styles for the TextField component.
+     * Custom styles for the container.
```

Comment:
It would be worth adding a NOTE here why we apply that to the container and not to the `TextField` component.

Reply:
Cool, thanks!